.vtex/catalog-info.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: store-image
+  description: VTEX components for dealing with images
+  tags:
+    - typescript
+    - react
+  annotations:
+    vtex.com/janus-acronym: ""
+    vtex.com/o11y-os-index: ""
+    grafana/dashboard-selector: ""
+    github.com/project-slug: vtex-apps/store-image
+    vtex.com/platform-flow-id: ""
+    backstage.io/techdocs-ref: dir:../
+    vtex.com/application-id: 9R38AHI2
+spec:
+  lifecycle: stable
+  owner: te-0013
+  type: frontend-ui
+  dependsOn: []
```

Comment:
In [other cases](https://github.com/vtex/messages/blob/master/.vtex/catalog-info.yaml#L13), we also have the `system: store-framework` property. Does it make sense here as well?